gitpod-io/gitpod
# When creating a new Stripe customer, also create a new Usage-Based Subscription for them

## Description


When creating a new Stripe customer, also create a new Usage-Based Subscription for them.

## Related Issue(s)

Finishes fixing https://github.com/gitpod-io/gitpod/issues/10327

## How to test


1. Create a team called "Gitpod" (or "Gitpod1", "Gitpod2", etc. -- first one that's not already taken)
2. Go to Team Settings > Team Billing
3. Wait for the Usage-Based Billing UI to show up
4. Upgrade to Usage-Based by adding a credit card (e.g. 4242 4242 4242 4242, 4/24, 424)
5. Once upgraded, click on "Manage Billing"
6. The Stripe Portal should show an active Subscription (linked to the Stripe Product "Gitpod Usage (with free tier)")

## Release Notes

```release-note
NONE
```

## Documentation
